js/piwik.js

```diff
@@ -4960,6 +4965,10 @@ if (typeof window.Matomo !== 'object') {
                 linkTrackingInstalled = false;
                 linkTrackingEnabled   = false;
             };
+            this.disableFingerPrint = function ()
```

Comment:
I wonder if we can find a better name here. Like `disableAccessBrowserFeatures` since someone could argue it's not a fingerprint that we create